src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -304,8 +304,10 @@ private void SolutionEvents_AfterClosing()
             }
         }
 
-        public void CleanCache()
+        public async Task CleanCacheAsync()
         {
+            await SolutionManager.CleanAsync();
```

Comment:
Why do you need this delegation? Consider moving the implementation here instead.